test/helper/clean.ts

```diff
@@ -47,20 +53,21 @@ const getOnePsy = (
     archived,
     state,
     adeli: `${getRandomInt()}829302942`,
-    address: `${getRandomInt()} SOLA 66110 MONTBOLO`,
+    address: `${getRandomInt()} ${addresses[Math.floor(Math.random() * 100) % 3]}`,
```

Comment:
To generate random values that seems real, we could use [faker](https://github.com/Marak/Faker.js)

Reply:
https://trello.com/c/qlmb3r34/159-use-faker-to-generate-mock-data